src/views/StoryViewer.vue

```diff
@@ -114,9 +159,9 @@ export default {
     background-color: #FFFFFF;
     animation-name: loading;
     animation-iteration-count: 1;
-    /* TODO - ideally the animation duration is
-    set as var related to frameDuration  */
-    animation-duration: 2s; 
+  }
+  .progress .paused {
+    animation-play-state: paused;
```

Comment:
TIL - never know this property! so nice 👍

Reply:
Yes! This was a discovery for me too